nipap/nipap/rest.py

```diff
@@ -45,6 +46,10 @@ def combine_request_args():
 
             if request_nipap_fullname:
                 args['auth'].update({'full_name': request_nipap_fullname})
+        elif request_authorization_header and \
+                request_authorization_header.startswith('Bearer'):
+            authoritative_source = {'auth': {'authoritative_source': 'jwt'}}
```

Comment:
The authoritative source is used to describe what system is the authoritative source of the data stored in NIPAP (such as a prefix) and doesn't really have that much to do with authentication. That shouldn't need any special handling for JWT auth.

Reply:
Är inte riktigt med på denna. Den behövs för att inte köra ut på rad 148.